.github/workflows/lint.yml

```
@@ -41,3 +41,28 @@ jobs:
         with:
           command:      fmt
           args:         --all -- --check
+  check-spellcheck:
+    name:               Check For Spelling and/or Grammar Mistakes
+    runs-on:            ubuntu-latest
+    env:
+      RUST_BACKTRACE:   full
+    steps:
+      - name:           Cancel Previous Runs
+        uses:           styfle/cancel-workflow-action@0.4.1
+        with:
+          access_token: ${{ github.token }}
+      
+      - name:           Checkout sources & submodules
+        uses:           actions/checkout@master
+        with:
+          fetch-depth:  5
+          submodules:   recursive
+      
+      - name:           Add cargo-spellcheck
+        run:            cargo install cargo-spellcheck
+      
+      - name:           Run spellcheck
+        uses:           actions-rs/cargo@master
+        with:
+          command:      spellcheck
+          args:         check  -m 1 -vv
```

```suggestion
          args:         check  -vv
```

Since the job is going to fail every time before we fix all of the errors I'd rather not mark every PR green, cause it's not clear how fast we can deal with it.

Ideally we would make a snapshot of all errors and just fail on newly introduced issues, but that might be too involved. I know you are planning to fix all errors anyway, hence I'm fine with having this job be kind of useless now, but at the very end let's switch to `-m 1`.

---

Yup, but I need to narrow source to analysis, try to do it like (in order to keep green checks):

`
search_dirs = ["./modules/currency-exchange"]
`

And to start correction from module `currency-exchange` but cargo-spellcheck does not take into consideration my way of thinking how to configure it ;/

---

I think `search_dirs` is more for extra dictionaries, not really for narrowing down the content. However looking at this example:
```
exec cargo spellcheck -m 99 $(git diff-index --cached --name-only --diff-filter=AM HEAD)
```
We can try to add modules here one by one, perhaps we can start with:
```
cargo spellcheck -m 1 -vv $(find modules/currency-exchange -name *.rs)
```
?